src/stdlib/format_timestamp.rs

```diff
@@ -32,6 +37,11 @@ impl Function for FormatTimestamp {
                 kind: kind::BYTES,
                 required: true,
             },
+            Parameter {
+                keyword: "tz",
```

Comment:
Somewhat of a nit, but I don't think we have a precedent for abbreviating parameter names. Could we call this `timezone`?

Reply:
Sure, no problem.